dodfminer/__version__.py

```diff
@@ -1,4 +1,4 @@
-version_info = (1, 2, 0)
+version_info = (1, 3, 6)
```

Comment:
Pode colocar a versão 1.3.7, lari? Eu tinha feito a tag 1.3.6 e aí quando for gerar uma tag tinha que dar um bump no patch.